### ISSUE TITLE
Apply Driver Pack process failed.

### ISSUE THREAD

**Issue by terraj01** [MSEndpointMgr/ModernDriverManagement#21]
i am running invoke-cmapplydriverpackage.ps1 4.0.2

invoke-cmapplydriverpackage.ps1 -PreCache -Endpoint "pri02.xxx.net" -TargetOSVersion "2004"

the applydriverpackage.log all looks successful it reads the service account name from the TS variable but it failed to apply driver package process failed, please refer to previous error or warning. i am not seeing any errors in the log so i am not sure what is failing.

**Comment by AlexanderZach** [comment]
Hello,

I'm facing the exact same issue.

Additional infos:

MECM version: 1910 inc. all hotfixes and rollups (Customer wants to wait for MECM 2006 Slow Ring)

DAT 6.4.8 Hotfix 3

DAT was previously configured and tested successfully using the legacy Webservice and changed to use the admin service (according to your and the Microsoft docs) & Invoke-Script Version 4.0.2

TS command:
invoke-cmapplydriverpackage.ps1 -BareMetal -Endpoint "xxx.xxx.xx" -TargetOSVersion 1909 (it made no difference putting the OS version in quotes or not)

SMSTS part:
<img width="946" alt="smstslog" src="https://user-images.githubusercontent.com/69917931/90779247-e2ece280-e2fd-11ea-8308-9062e2b0e088.png">

ApplyDriverPackage.Log:

<img width="707" alt="ApplyDriverPackageLogPart1" src="https://user-images.githubusercontent.com/69917931/90774132-40c9fc00-e2f7-11ea-80ad-01643643a7b1.png">
<img width="700" alt="ApplyDriverPackageLogPart2" src="https://user-images.githubusercontent.com/69917931/90774140-4293bf80-e2f7-11ea-8de1-508ac25a3238.png">


Testing the driver installation in debug mode on a different client also failed with the same error message!

As an alternative we also tested the legacy webservice step and the legacy scripts 3.0.4 (which was used before switching to version 4) - same ERROR
Test version 3.0.8 - same ERROR

We tested two different devices and none of them was working:

Lenovo T590
Lenovo T15 Gen 1

best regards
Alex

**Comment by terraj01** [comment]
I guess I’m glad I am not the only one seeing this.

**Comment by NickolajA** [comment]
Could you verify that all of your packages have at least a single SKU (4 chars) in the comments property? E.g. (Models included:1234)

**Comment by CarValScott** [comment]
Yes mine all have the SKU's listed in the comments. With running the debug scripts, I do find that the Dell Latitudes 5310's also gets the 5300 driver package. When deleting the 5300 drive package the debug script see's only one, but we still get the errors as listed above.  

here is the WinPE log file. 

![ts2](https://user-images.githubusercontent.com/47256082/93132824-e7f24700-f69b-11ea-8aba-b6ccd31f381c.PNG)

Here is the debug log.

![ts1](https://user-images.githubusercontent.com/47256082/93131928-6c43ca80-f69a-11ea-9e91-a9865ccea81a.PNG)

**Comment by nodiaque** [comment]
Verify something, is there space somewhere in the skus in the package? I had a similar problem, my script would just end nowhere no error. I had a package that had a space in the comment and because of that, it would kill the process. That might trigger a mistmatch here. Maybe try recreate the package with the latest driver automation tool?
